service/src/main/java/com/commercetools/pspadapter/payone/config/ServiceConfig.java

```diff
@@ -25,6 +28,7 @@
     private final String scheduledJobCronLongTimeFrame;
     private final String applicationName;
     private final String applicationVersion;
+    private final List<String> personalDataToRemove;
```

Comment:
Why is it a list and not simple boolean? Hard to imagine that one would set it for one tenant only.

Reply:
You can define multiple fields as personal. Why not?

Reply:
The question is here if it is necessary to explicitly define those fields or they are obvious and can be hard coded as constants?